user/views.py

```diff
@@ -18,7 +17,7 @@ class CreateTokenView(ObtainAuthToken):
 
 class ManageUserView(generics.RetrieveUpdateAPIView):
     serializer_class = UserSerializer
-    authentication_classes = (TokenAuthentication,)
+    # authentication_classes = (TokenAuthentication,)
```

Comment:
The `authentication_classes` line is commented out. If token authentication is required for the `ManageUserView`, consider uncommenting this line to enforce it. Ensure consistency with the authentication strategy used across the application.